source/api_documentation/call_ingestion_api/index.rst

```diff
@@ -0,0 +1,321 @@
+--------------
+
+Remember to check the HTTP status code returned. This helps greatly when debugging.
+
+.. list-table::
+  :widths: 8 40
+  :header-rows: 1
+  :class: parameters
+
+
+  * - Status Code
+    - Meaning
+
+  * - 201 Created
+    - A new call creation request was successfully created.
+
+  * - 202 Accepted
+    - The call creation request has already been received and is currently being processed.
+
+  * - 400 Bad Request
+    - Attempted to make a request with an invalid API Version for route. Check the error message for any neccessary corrections
+
+  * - 401 Not Authorized
+    - Invalid or missing oauth token.
+
+  * - 403 Forbidden
+    - Attempted to access an invalid resource or provided invalid data. Check the errors object in the response.
+
+  * - 409 Conflict
+    - The **external_call_unique_id** in the request has already been used.  Please contact the Invoca support team at questions@invoca.com for further assistance.
+
+Request Parameters
+------------------
+
+Passed in “application/json” format.
+
+**Call Parameters**
+
+These are the call datails used when creating the call in the Invoca platform.
+
+    **Required**
+
+    `external_call_unique_id` The unique ID of the call in the system fom the external system.  This field is required to be unique across all calls within a network that are submitted from external sources.
+
+    `start_time` This is the date and time when the call started on the call origination platform.  Please see the **Timestamp Formats** section below for descriptions of supported timestamps.
+
+    `destination_phone_number` DNIS in E.164 format +country national_number; example: ‘+18885551212’. UK and Spanish numbers are also supported. Their country codes are +44 and +34 respectively.
+
+    `calling_phone_number` ANI in E.164 format +country national_number; example: ‘+18885551212’.
+
+    `advertiser_campaign_id_from_network` ID from network field on advertiser campaign; only calls within this advertiser campaign will be considered (campaign must be in the organization accessed via API).
+
+    `call_direction` The direction of the call flow.  Accepted values: *inbound* or *outbound*.
+
+    `recording_url` The URL to the call recording. Please see the **Supported Recording Formats** and **Supported Recording Access Options** sections for more details.
+    
+-----
+
+**Signal Parameters**
+
+Used to create the fields of a signal. The Signal name provided in a request **must** already exist in your **Signal AI configuration**.
+
+    **Required**
+
+    `name` The name describing the signal event. For reporting a sale happened on a call, “Sale” is recommended.
+    Other examples include “Free Trial”, “2yr Subscription”, “Cancellation.”
+    This can be used elsewhere in the system and should be a small list of values meaningful to your organization.
+    Names are matched case-insensitively, but we will preserve and use the casing of the first time the signal name is reported.
```

Comment:
We should drop "but we will preserve and use the casing of the first time the signal name is reported" since we removed the lazy-create feature